_posts/2016-12-14-thanks-to-greg.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+subheadline: "Gratitude"
+title: "Congratulations to Greg on his new position!"
+teaser: "Thanks for all his amazing and inspirational work"
+header:
+    image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+    - blog
+comments: true
+show_meta: true
+authors: ["Data Carpentry Steering Committee and Staff"]
+---
+
+We’re excited for [Greg’s new position as a Computer Science Education Lead at Shopify](https://software-carpentry.org/blog/2016/12/next-steps.html), where he will help with their CS education partnerships and are enthusiastic to see where his creativity, passion, commitment and hard work take him next. There aren’t words to express the gratitude we have for everything Greg has done and his tireless work on behalf of better research software practices that have an impact on how we do research everyday. While this was the initial goal, along the way Greg created an ethos of collaborative and open lesson development, of teaching shared curriculum, of learning to be a better teacher and most of all he created a community of people who are excited to work together. He made those of us who felt alone in our labs working in programming or data analysis feel like we weren’t alone and that we were connected to other people who shared our enthusiasm for data and code and our passion to share that perspective with others. When we did a survey of instructors, the most common reason people said they wanted to become instructors is because they talked to Greg. Greg has collectively and individually inspired over 700 instructors that we have today as well as countless learners in any of the workshops that he’s taught. He’s made us feel like we can do more than we thought and that we had something important to contribute. For Data Carpentry, he is the one who encouraged us to scale it beyond an initial few workshops and make it a Carpentry and was on the Steering Committee for the crucial time of getting it off the ground. We’re sure there are so many other stories from people who Greg inspired to be bold and create something new and go after something that was valuable but difficult.
```

Comment:
I'd suggest we start not assuming that everyone know's perfectly well who _Greg_ is and what's happening with him. For example: "_As many of you will know by now, Greg Wilson, Director of Instructor Training for Software Carpentry (and thus indirectly also for Data Carpentry), has announced that he will step down from his position at the end of January 2017 to join Shopify as their Computer Science Education Lead. We're very excited ..."_